doc/api/esm.md

```diff
@@ -1086,6 +1086,9 @@ The resolver can throw the following errors:
 >          **PACKAGE_EXPORTS_RESOLVE**(_packageURL_, _packageSubpath_,
 >           _pjson.exports_, _defaultConditions_).
 >    1. Otherwise, if _packageSubpath_ is equal to _"."_, then
+>       1. If _pjson_ is not **null** and _pjson_._type_ is equal to _"module"_,
+>          then,
+>          1. Throw a _Package Path Not Exports_ error.
```

Comment:
This should be updated to match <https://github.com/nodejs/node/commit/6741cc2149f92b12f8640ddd07d6e5646bc190cb>.

Reply:
Thanks, I'll leave out the spec change for now since its a deprecation.